crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -247,6 +260,13 @@ impl AuthorityPerEpochStore {
             end_of_publish: Mutex::new(end_of_publish),
             pending_consensus_certificates: Mutex::new(pending_consensus_certificates),
             wal,
+            tallying_rule_score_metric: register_int_gauge_vec_with_registry!(
```

Comment:
I think it may be problematic if we try to register the same metric multiple times, which would happen since we recreate the epoch store on every epoch

Reply:
Right. I'll see if I can move this to AuthorityMetrics.